client-app/modules/quotes/pages/view-quote.vue

```diff
@@ -62,7 +62,7 @@
             <div class="flex items-center gap-2">
               <span class="text-base font-bold">{{ $t("quote_details.status") }}:</span>
 
-              <QuoteStatus class="min-w-[7.785rem]" :status="quote.status" />
+              <QuoteStatus class="min-w-0" size="sm" :status="quote.status" />
```

Comment:
you sure about min-w-0? Could it collapse?

Reply:
yes, i'm sure. flex hack for truncate.